deployments/mainnet/Solana_Adapter.json

```diff
@@ -328,29 +328,29 @@
       "type": "function"
     }
   ],
-  "transactionHash": "0x940c1e4bf26e94e657bc1e1344d3c2aea44e80a051837b75812283668a23c3a7",
+  "transactionHash": "0x5256d567a2ce4cbc96f04cb3b0885fad53fd48d163b67a79d9dfeaaba21f6fc3",
   "receipt": {
     "to": null,
     "from": "0x9A8f92a830A5cB89a3816e3D267CB7791c16b04D",
-    "contractAddress": "0x1E22A3146439C68A2d247448372AcAEe9E201AB1",
-    "transactionIndex": 72,
+    "contractAddress": "0x9F788694934fD2Ed34D5340B9a76EB34f2bFD7B3",
+    "transactionIndex": 22,
     "gasUsed": "1034334",
     "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-    "blockHash": "0x66dae79469e26570eb7960c9b1daa884ca592e1085f00879f1012d1e5e309022",
-    "transactionHash": "0x940c1e4bf26e94e657bc1e1344d3c2aea44e80a051837b75812283668a23c3a7",
+    "blockHash": "0x60c1b45973d5aba6a92d540aa1c8b879817b8a1a3cafcb9b20ce4ba474c44dd1",
+    "transactionHash": "0x5256d567a2ce4cbc96f04cb3b0885fad53fd48d163b67a79d9dfeaaba21f6fc3",
     "logs": [],
-    "blockNumber": 22566473,
-    "cumulativeGasUsed": "5278156",
+    "blockNumber": 22595936,
+    "cumulativeGasUsed": "4109786",
     "status": 1,
     "byzantium": true
   },
   "args": [
     "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
     "0xBd3fa81B58Ba92a82136038B25aDec7066af3155",
     "0x0a992d191deec32afe36203ad87d7d289a738f81",
-    "0xff09aa2d3eb1bc9da19e82264930e13d3993e1160f3039ef21afd1565376efca",
+    "0xb7664086de37ee70821c10445b162f2c7ec8795bd0800c1462949e2328d1dd5a",
     "0xc6fa7af3bedbad3a3d65f36aabc97431b1bbe4c2d2f6e0e47ca60203452f5d61",
-    "0xe11ef405ee6e9a6497fb8209d04040216281c57264730415648de979f6cec7fa"
+    "0xf5d9ddc2b5d994277e15ea380117a3f8ef04ce1e37e2c678c3be4d11b2a5d034"
```

Comment:
decodes to USDC vault for the new `svm_spoke` at [HYhZwefNFmEm9sXYKkNM4QPMgGQnS9VjC6kgxwrGk3Ru](https://explorer.solana.com/address/HYhZwefNFmEm9sXYKkNM4QPMgGQnS9VjC6kgxwrGk3Ru)